addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/handler/LoxoneMiniserverHandler.java

```diff
@@ -249,6 +253,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
                 return;
             }
+
+            if (control instanceof LxControlColorPickerV2) {
+                LxControlColorPickerV2 colorPicker = (LxControlColorPickerV2) control;
+
+                if (command instanceof HSBType) {
```

Comment:
According to openHAB docs, Color item can take commands: OnOff, IncreaseDecrease, Percent, HSB. We need to handle those, additonally to HSB.